tests/features/piwik.feature

```diff
@@ -27,7 +27,10 @@ Feature: Check Piwik
     And the response should contain the meta tag with the "X-UA-Compatible" name and the "IE=edge" content
 
   Scenario: Check if the PIWIK script flags forbidden pages
-    Given I go to "ecaslogout"
+    Given these modules are enabled
+      | modules            |
+      | ecas_env           |
```

Comment:
ECAS_env is enabled by default in the platform when we install it with "install-platform" target. The module state are stored in the DB.

So, what is the relationship between the static variables clearing and the need to enable explicitly "ECAS_env" for this scenario; especially when we do not do it for the `ecas.feature` ones?

I raise the point because this step force the site settings that are set correctly in the default platform ones. We change then the test parametrising where it is not requested.

Reply:
Again.. ecas_env needs to be enabled because of communities.